img/ImageComponents.ts

```diff
@@ -12,6 +12,7 @@ import ArrowUpHoverImage from "./svg/arrow-up_hover.svg";
 import ArrowUpHoverHighContrastImage from "./svg/arrow-up_hover-highcontrast.svg";
 import ArrowWithLineDownBlueImage from "./svg/arrow-with-line__down--blue.svg";
 import ArrowWithLineUpBlueImage from "./svg/arrow-with-line__up--blue.svg";
+import BlaattDokumentImage from "./svg/blaattdokument.svg";
```

Comment:
Vi lar vanligvis å=a. Så for å være konsistent er vel BlattDokumentImage bedre. Men jeg vet ikke om det var så forståelig. Så kanskje engelsk er løsninga, BlueDocumentImage?

Reply:
Ja, kan endre.